content/push/device.textile

```diff
@@ -40,6 +49,14 @@ h3. Configure an iOS device
 * Go to *push notifications setup*, click *configure push*.
 * Add your *.p12* file.
 
+h4. Add Ably Flutter to your iOS project
```

Comment:
If there isn't a specific push package required then this is a step that we aren't doing for Swift / Kotlin already and is just generic Flutter SDK setup. Do you think this should be added for all, or excluded?